components/utils.py

```diff
@@ -1104,13 +1150,33 @@ def _list_executions_with_retries(headers, execution_id, retries=6):
     ctx.abort_operation(err)
 
 
-def _create_maintenance_headers(upgrade_props=True):
+def create_maintenance_headers(upgrade_props=True):
     headers = {'X-BYPASS-MAINTENANCE': 'True'}
     auth_props = get_auth_headers(upgrade_props)
     headers.update(auth_props)
     return headers
 
 
+def is_sanity_dep_exist(manager_ip, dep_id):
```

Comment:
Um.. all functions you added under utils.py are used only once in the sanity script.. so why are they utils?